modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -20,7 +20,7 @@ You can create a virtual machine (VM) snapshot for an offline or online VM by cr
 +
 [source,yaml]
 ----
-apiVersion: snapshot.kubevirt.io/v1beta1
+apiVersion: snapshot.kubevirt.io/v1alpha1
```

Comment:
**QE REVIEW NEEDED** Please verify this change.

Reply:
it is correct